src/core/scene/a-scene.js

```diff
@@ -118,7 +118,12 @@ module.exports.AScene = registerElement('a-scene', {
         window.addEventListener('vrdisplaypresentchange', this.onVRPresentChangeBound);
 
         // Enter VR on `vrdisplayactivate` (e.g. putting on Rift headset).
-        window.addEventListener('vrdisplayactivate', function () { self.enterVR(); });
+        window.addEventListener('vrdisplayactivate', function () {
```

Comment:
Checking `this.effect` may work, but not sure whether subsequent event listener will work, as `vrdisplayactivate` initiation context would be lost; so far, firing next tick seems to be long enough for the renderer startup to happen, as that seems long enough for `render-target-loaded` handling to propagate.